### PR TITLE
test(e2e) add multizone external test

### PR DESCRIPTION
**Pull request by nickolaev**
### Summary

Add external service test with Multizone deployment.
Make sure that the IPv6 addresses are not used in Universal unless requested.

### Documentation

- [ ] Not needed

### CODE REVIEW

File: test/framework/setup.go [hunk 1]
```diff
@@ -275,7 +275,8 @@ func EchoServerUniversal(name, mesh, echo, token string, fs ...DeployOptionsFunc
 func IngressUniversal(mesh, token string) InstallFunc {
 	return func(cluster Cluster) error {
 		uniCluster := cluster.(*UniversalCluster)
-		app, err := NewUniversalApp(cluster.GetTesting(), uniCluster.name, AppIngress, AppIngress, true, []string{})
+		isipv6 := IsIPv6() // should we check opts as everyone else does?
```
Comment (jakubdyszkiewicz) [inline test/framework/setup.go:278]: what's the story here?

Should we remove WithIPv6 then if we cannot override the setting from env var?
(I see this this was the only usage)

File: test/framework/universal_app.go [hunk 2]
```diff
@@ -206,6 +206,9 @@ func NewUniversalApp(t testing.TestingT, clusterName, dpName string, mode AppMod
 		opts.OtherOptions = append(opts.OtherOptions, "--cap-add", cap)
 	}
 	opts.OtherOptions = append(opts.OtherOptions, "--network", "kind")
+	if !isipv6 {
+		opts.OtherOptions = append(opts.OtherOptions, "--sysctl", "net.ipv6.conf.all.disable_ipv6=1")
```
Comment (jakubdyszkiewicz) [inline test/framework/universal_app.go:210]: a little comment why we need it?

File: test/framework/setup.go (hunk 1 above)
Comment (nickolaev) [reply test/framework/setup.go:278]: The INgress Setup does not have opts today. WithIPv6 will be removed indeed.

**Review by jakubdyszkiewicz — APPROVED** [review]

**Comment by jakubdyszkiewicz** [comment]
@mergify update

**Pull request merged**